.gitignore

```diff
@@ -1,3 +1,5 @@
 .vs
 bin
 obj
+build
+/build
```

Comment:
You shouldn't need both of these, the one on line 5 is already covered by the one on line 4.

Reply:
done